src/TwigComponent/doc/index.rst

```diff
@@ -295,6 +295,18 @@ prefix the attribute with ``:`` or use the normal ``{{ }}`` syntax:
     // pass object, array, or anything you imagine
     <twig:Alert :foo="['col' => ['foo', 'oof']]" />
 
+Boolean props require using the dynamic syntax:
```

Comment:
Instead of focusing just on bool, maybe we can do a full example of the dynamic syntax, with scalar, bool, array, and objects, ...

Reply:
I think that the dynamic syntax is already explained for objects, arrays, expressions, etc.

The problem is that when using scalar values, only booleans are problematic: `someProp="foo"` (`foo` is a string in PHP), `someProp="5"` (will be casted to `(int) 5` in PHP) but `someProp="false" will be casted to `true` in PHP because it's not parsed as a proper boolean value.

That was my reasoning to mentioning only this. Thanks.

Reply:
Ha yes ok thanks @javiereguiluz for the explanation!